tests/openwrt/test_interfaces_dsa.py

```diff
@@ -791,11 +786,11 @@ def test_parse_simple_bridge(self):
     _complex_bridge_uci = """package network
 
 config interface 'eth0'
-    option ifname 'eth0'
+    option device 'eth0'
```

Comment:
we'll need to supply this device definition, applies for all the other occurrences